spec/features/admin_user/login_and_logout_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.feature "the admin user" do
+  let(:admin) { User.create(full_name: "Admin Annie",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.